libnopegl/src/text_external.c

```diff
@@ -60,9 +60,9 @@ static int load_font(struct text *text, const char *font_file)
     FT_Face ft_face = NULL;
     hb_font_t *hb_font = NULL;
 
-    FT_Error ft_error = FT_New_Face(text->ctx->ft_library, font_file, 0, &ft_face);
+    FT_Error ft_error = FT_New_Face(text->ctx->ft_library, font_file, face_index, &ft_face);
```

Comment:
I think we should sanitize face_index against negative values, see: https://freetype.org/freetype2/docs/reference/ft2-face_creation.html#ft_open_face.

Reply:
Added a `[0,0xffff]` range check at fontface init